tests/lite/test_lite.py

```diff
@@ -444,25 +465,3 @@ def run(self):
             assert self.is_global_zero == (self.local_rank == 0)
 
     Lite(strategy=DeepSpeedPlugin(stage=3, logging_batch_size_per_gpu=1), devices=2, accelerator="gpu").run()
-
-
-def test_replace_dataloader_init_method():
-    """Test that the context manager enables to save the parameters passed to the DataLoader __init__ method."""
-
-    class CustomDataLoader(DataLoader):
-        def __init__(self, extra_argument: int, *args, **kwargs):
-            super().__init__(*args, **kwargs)
-
-    dataloader = CustomDataLoader(extra_argument=1, dataset=range(1))
-    lite = EmptyLite()
-    with pytest.raises(MisconfigurationException, match="extra_argument"):
```

Comment:
reviewer: moved this part of the test to a separate test above `test_setup_dataloaders_raises_for_unknown_custom_args`,
the rest is already tested in `test_setup_dataloaders_with_custom_type` added in #10334